src/scancodes.rs

```diff
@@ -418,7 +419,9 @@ impl ScancodeSet for ScancodeSet2 {
             0x74 => Ok(KeyCode::ArrowRight),   // E074
             0x75 => Ok(KeyCode::ArrowUp),      // E075
             0x7A => Ok(KeyCode::PageDown),     // E07A
+            0x7C => Ok(KeyCode::PrintScreen),  // E07C
             0x7D => Ok(KeyCode::PageUp),       // E07D
+            0x7E => Ok(KeyCode::ControlBreak), // E07E
```

Comment:
Perhaps this should be `Break` or `ControlPause`? `Pause` is what you get if you don't press Ctrl with it, right?